src/Microsoft.AspNet.Identity.EntityFramework/UserStore.cs

```diff
@@ -940,7 +961,7 @@ public virtual Task<bool> GetTwoFactorEnabledAsync(TUser user, CancellationToken
                         && userclaims.ClaimType == claim.Type
                         select user;
 
-            return (IList<TUser>)await query.ToListAsync(cancellationToken);
+            return await query.ToListAsync(cancellationToken);
```

Comment:
AFAIK the conversion issue is only between Task<List<T>> and Task<IList<T>>. Once the List<T> is unwrapped there shouldn't be a problem.